main.py

```diff
@@ -275,11 +275,9 @@ def ui_access_wrapper(self, *args, **kwargs):
         learner_playlist.LearnerPlaylistHandler),
 
     get_redirect_route(
-        r'/assetsdevhandler/<exploration_id>/assets/image/<encoded_filename>',
-        resources.ImageDevHandler),
-    get_redirect_route(
-        r'/assetsdevhandler/<exploration_id>/assets/audio/<encoded_filename>',
-        resources.AudioDevHandler),
+        r'/assetsdevhandler/<exploration_id>/'
+        'assets/<asset_type:(image|audio)>/<encoded_filename>',
```

Comment:
Oh, cool! I didn't know we had this ability. That might obviate the need for the asset_type check above.

Can you add backend tests to verify the image, audio, and invalid cases?

Reply:
I can definitely add one for invalid asset.
Do we still want to add image and audio? I think there're already tests covering this in resources_test.py

Reply:
Ah, if there's an existing test, it's fine. Just make sure there exist tests in the codebase for all three cases.

Thanks!

Reply:
Sounds good!